### PR TITLE
Bug 1368107 - Increase the amount of items per page.

### PR DESCRIPTION
**Pull request by josejulio**
This increases the number of items per page on every gui/portal-war
that uses the PageControlSettingsUIBean.

### CODE REVIEW

**Comment by josejulio** [comment]
Not sure if this is the proper way to do it.

The user needs to add `rhq.server.gui.max-items-per-page` property to `rhq-server.properties` file.
Also this will propagate to every UI that uses `PageControlSettingsUIBean`.
If `rhq.server.gui.max-items-per-page` is not set, it will use the default values.

Do you consider that I should only modify the requested view ? (Content -> New)
(it seems) that can be done as easily as this.
@spinder @burmanm

File: modules/enterprise/gui/portal-war/src/main/java/org/rhq/enterprise/gui/common/paging/PageControlSettingsUIBean.java [hunk 1]
```diff
@@ -24,8 +24,25 @@
...
-    private SelectItem[] pageSizes = new SelectItem[] { new SelectItem("15", "15"), new SelectItem("30", "30"),
-        new SelectItem("45", "45") };
+    private SelectItem[] pageSizes;
+    private static String MAX_ITEMS_PER_PAGE = "rhq.server.gui.max-items-per-page";
+
+    public PageControlSettingsUIBean() {
+        try {
+            String maxItemsPerPage = System.getProperty(MAX_ITEMS_PER_PAGE);
+            if (maxItemsPerPage != null && Integer.parseInt(maxItemsPerPage) > 45) {
+                pageSizes = new SelectItem[] { new SelectItem("15", "15"), new SelectItem("30", "30"),
```
Comment (pilhuhn) [inline modules/enterprise/gui/portal-war/src/main/java/org/rhq/enterprise/gui/common/paging/PageControlSettingsUIBean.java]: I think I would expect that those 15/30/45 would scale with the max and be at 1/4, 1/2 and 3/4 of the max

Comment (burmanm) [inline modules/enterprise/gui/portal-war/src/main/java/org/rhq/enterprise/gui/common/paging/PageControlSettingsUIBean.java]: Do we still have those UI pages where the scrolling starts to be buggy if there's more than X items per page? I'm not sure if all of those were fixed..

Comment (josejulio) [inline modules/enterprise/gui/portal-war/src/main/java/org/rhq/enterprise/gui/common/paging/PageControlSettingsUIBean.java]: Uhg, I'm not sure. I might have to test with a ton of items.
I recall seeing in the code a bug when we had lots of pages. Not sure if you are refering to that one. https://github.com/josejulio/rhq/blob/d5dfe2ed9d42af51c6c3d3793f79e315c52ed092/modules/enterprise/gui/portal-war/src/main/webapp/rhq/resource/include/pagination-plain.xhtml#L66-L66

**Pull request merged**